tests/test_interfaces.py

```diff
@@ -14,25 +14,18 @@
 
 import unittest
 from random import randint
-import jsonschema
-
-from factoring.interfaces import factor
-from factoring.json_schema import json_schema
 
+from demo import factor
 
 class TestInterfaces(unittest.TestCase):
 
-    def test_factor_output(self):
-        P = randint(0, 2**6-1)
-        output = factor(P)
-        jsonschema.validate(output, json_schema)
-
     def test_factor_invalid(self):
         for P in [-1, 64, 'a']:
             self.assertRaises(ValueError, factor, P)
 
-    @unittest.skip("Not really a unittest")
     def test_factor_validity(self):
         for P in [12, 21, 49]: # {a*b for a in range(2**3) for b in range(2**3)}:
-            output = factor(P)
-            self.assertTrue(output['results'][0]['valid'])
+            retries = 3
+            while retries > 0 and not factor(P)['Results'][0]['Valid']:
+                retries -= 1
+            self.assertGreater(retries, 0)
```

Comment:
How do you feel about implementing this with the `@dwave.cloud.utils.retried(3)` decorator?

Reply:
I feel excited and honored, sir.

Reply:
I couldn't possible ask for anything more.

Reply:
In that case, maybe you'll be good enough to merge this PR.